test/extended/operators/images.go

```diff
@@ -117,12 +117,18 @@ var _ = Describe("[sig-arch] Managed cluster", func() {
 					invalidPodContainerImagePullPolicy.Insert(fmt.Sprintf("%s/%s/%s imagePullPolicy=%s", pod.Namespace, pod.Name, container.Name, container.ImagePullPolicy))
 				}
 			}
+
+			if pod.Status.Phase != v1.PodRunning {
+				continue
+			}
 			// check if the container's image from the downstream.
 			for j := range pod.Spec.Containers {
 				containerName := pod.Spec.Containers[j].Name
 				commands := []string{
 					"exec",
 					pod.Name,
+					"-n",
+					pod.Namespace,
```

Comment:
This certainly doesn't hurt :+1:

Reply:
I didn't notice `oc.SetNamespace(pod.Namespace)` here, now I fixed the following error from `SetNamespace` function.
```
Feb  7 06:40:32.129: INFO: unable to run command:[exec service-ca-operator-bbb9b5cf7-29z7g -c service-ca-operator -- cat /etc/redhat-release] with error: exit status 1
Feb  7 06:40:32.130: INFO: Running 'oc --kubeconfig=/home/piqin/flexy_output/flexy/workdir/install-dir/auth/kubeconfig exec service-ca-8676df98ff-4hvkv -c service-ca-controller -- cat /etc/redhat-release'
Feb  7 06:40:32.762: INFO: Error running /usr/bin/oc --kubeconfig=/home/piqin/flexy_output/flexy/workdir/install-dir/auth/kubeconfig exec service-ca-8676df98ff-4hvkv -c service-ca-controller -- cat /etc/redhat-release:
StdOut>
Error from server (NotFound): pods "service-ca-8676df98ff-4hvkv" not found
StdErr>
Error from server (NotFound): pods "service-ca-8676df98ff-4hvkv" not found

```